omnigan/data.py

```diff
@@ -81,6 +81,11 @@ def pil_image_loader(path, task):
         arr = arr.astype(np.float32)
         arr[arr != 0] = 1 / arr[arr != 0]
 
+    if task == 'm':
+        arr[arr != 0] = 1 / arr[arr != 0]
+        #Make sure mask is RGB for the sake of transforms
```

Comment:
depth loads inverse values, masks don't

Reply:
True. I meant to binarize the masks here.